source/_integrations/netgear.markdown

```diff
@@ -17,15 +18,41 @@ ha_codeowners:
 ha_ssdp: true
 ---
 
-This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device.
+This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device and control the NETGEAR device.
+Both routers and access points can be used with this integration. Some access points will not be automatically discovered and need to be set up manually.
 
 {% include integrations/config_flow.md %}
 
-Most NETGEAR routers use port 5000 to communicate, however the following list of models are known to use port 80:
-- Nighthawk RAX50
-- Nighthawk X4S - AC2600 (R7800)
-- Orbi
-- XR500
-When setup through ssdp discovery the port schould be automatically detected.
+## Optional settings
+The options flow of the NETGEAR integration allows you to tweak the behavior of the NETGEAR integration.
 
-The options flow of the NETGEAR integration (in the sidebar of your Home Assistant instance click on "Configuration" -> "Integrations" -> find the NETGEAR integration and click "Configure") allows you to specify the 'consider_home' time. This is the amount of seconds to wait till marking someone as not home after not being seen. This parameter is most useful for households with Apple iOS devices that go into sleep mode while still at home to conserve battery life. iPhones will occasionally drop off the network and then re-appear. consider_home helps prevent false alarms in presence detection.
+### Consider_home
```

Comment:
```suggestion
### Consider_home

```
After each header, first a white line and then text

Reply:
done